unison-src/transcripts/diff-namespace.output.md

```diff
@@ -245,36 +240,38 @@ unique type Y a b = Y a b
     
     17. X    18. X' (added)
 
-.> alias.type ns1.X ns1.X2
+scratch/ns1> alias.type X X2
 
   Done.
 
-.> alias.type ns2.A' ns2.A''
+scratch/ns2> alias.type A' A''
 
   Done.
 
-.> fork ns2 ns3
+scratch/ns2> branch /ns3
 
-  Done.
+  Done. I've created the ns3 branch based off of ns2.
+  
+  Tip: To merge your work back into the ns2 branch, first
+       `switch /ns2` then `merge /ns3`.
 
-.> alias.term ns2.fromJust' ns2.yoohoo
+scratch/ns2> alias.term fromJust' yoohoo
 
   Done.
 
-.> delete.term.verbose ns2.fromJust'
+scratch/ns2> delete.term.verbose fromJust'
 
   Name changes:
   
-    Original            Changes
-    1. ns2.fromJust  ┐  2. ns2.fromJust' (removed)
-    3. ns2.fromJust' │  
-    4. ns2.yoohoo    │  
-    5. ns3.fromJust  │  
-    6. ns3.fromJust' ┘  
```

Comment:
We're running the delete from inside a specific branch now, so it doesn't show Originals from the others.